src/main/java/uk/nhs/tis/sync/message/publisher/RabbitMqTcsPmUpdatePublisher.java

```diff
@@ -0,0 +1,32 @@
+package uk.nhs.tis.sync.message.publisher;
+
+import java.util.List;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * publish messages to rabbitMq queues.
+ */
+@Component
+public class RabbitMqTcsPmUpdatePublisher {
+
+  @Value("${application.rabbit.reval.exchange}")
+  private String exchange;
+
+  @Value("${application.rabbit.reval.routingKey.currentpm.nightlysync}")
+  private String routingKey;
+
+  @Autowired
+  private RabbitTemplate rabbitTemplate;
```

Comment:
Not certain but as the config containing the `RabbitTemplate` is dependant on a property, this and/or the use of it might need to be too.

Reply:
Sorry, I don't quite get your concern here.
I checked the publisher in revaidation-recommendation service, but I couldn't see any benefits of abstracting an interface (like revalidation-recommendation service does). Not sure if this is something you might think is better.

Reply:
The RabbitConfig includes `@ConditionalOnProperty("spring.rabbitmq.host")`.  So if that property isn't provided, is rabbitTemplate null or configured with a default that causes an exception in `publishToBroker()`?

Reply:
Ah, this rabbitConfig was copied from tcs. I will remove this annotation.

Reply:
Done.